libs/blocks/aside/aside.css

```diff
@@ -585,24 +595,16 @@
   stroke: var(--color-white);
 }
 
-.aside.promobar.popup .promo-text .text-area {
-  margin-top: var(--spacing-m);
-}
-
-.aside.promobar.popup .promo-text .icon-area + .text-area {
-  margin-top: 0;
-}
-
 @media screen and (max-width: 600px) {
   .aside.no-media:not(.notification) .foreground.container {
     gap: 0;
   }
 
   .aside.split.no-media:not(.notification) .foreground.container .text {
     padding: var(--spacing-xxxl) 0;
-  }  
+  }
 
-  .aside.promobar .mobile-up.hide-block ~ .promo-close {
+  .aside.promobar .mobile-up.hide-block~.promo-close {
```

Comment:
Is the close button available for simple promobars or just the `popup` variant ones? Would it make sense to make it specific to the `popup` variant if that is the case?

Reply:
@overmyheadandbody I added the popup variant to these